config/listflag.go

```diff
@@ -34,8 +34,13 @@ func (lf *listFlag) Set(value string) error {
 		return nil
 	}
 
-	lf.value = value
-	lf.values = strings.Split(value, lf.sep)
+	if value == "" {
```

Comment:
this case is all defaults, and you should better test for only the `!=` case and put the code from the else branch here.

Reply:
Do you suggest to just reverse the condition or omit else branch?
It should reset previous value so we'll have else branch anyway.

Reply:
If it needs to reset, it's fine like that